src/server/req_modify.c

```diff
@@ -284,6 +289,32 @@ req_modifyjob(struct batch_request *preq)
 				outsideselect = prsd;
 			}
 		}
+		if (i == (int) JOB_ATR_resc_released) {
+			if (jt == IS_ARRAY_Single) {
```

Comment:
That's a good catch. We should reject in all other cases. I'll add an else block.